src/pages/favorites.jsx

```diff
@@ -0,0 +1,23 @@
+import React, { useEffect } from 'react';
+import PageTitle from '../atoms/pageTitle';
+import Cardlist from '../molecules/cardlist';
+import store from '../Redux/store';
+
+const Favorites = () => {
+  // eslint-disable-next-line
+    useEffect (()=> {window.scrollTo(0,0)},[]);
```

Comment:
Hmmm, you shouldn't need this line. Pretty sure the react router is supposed to handle this.